Themes/default/css/index.css

```diff
@@ -247,25 +436,15 @@ a.new_posts:visited {
 	clear: both;
 }
 .clear_left {
-	clear: left;
+	clear: inline-start;
 }
 .clear_right {
-	clear: right;
+	clear: inline-end;
 }
 .reset {
 	all: unset;
 	outline: revert;
 }
-/* This is about links */
-a, a:visited, .reset.link {
-	color: var(--link);
-	text-decoration: none;
-	transition: ease 0.35s;
-}
-a:hover, .reset.link:hover {
-	text-decoration: underline;
-	cursor: pointer;
-}
```

Comment:
The idea here is to make button elements with the class `reset` appear as hyperlinks. See the `code` bbc  to see it in action, added via JavaScript.

The link class makes it more specific so that any leftover scripts from the 2.0 era don't accidentally get it applied to lists.

Reply:
Good to know, I'll add some of that context to a comment.

Reply:
I just checked and this is intact, it just moved to the top.

And with
``
.reset {
	all: unset;
	outline: revert;
}
``
It seems fine on code buttons.
It think it just needs to be .reset, button.reset to raise the specificity.